_field-types/mapping-parameters/doc-values.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: doc_values
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 25
+has_children: false
+has_toc: false
+---
+
+# Doc_values
+
+OpenSearch indexes most fields by default for searching. While the inverted index allows quick term-to-document lookups, doc values enable efficient document-to-term access for sorting, aggregations, and scripting.
```

Comment:
Is this specific to keyword or any string related fields where doc values is not supported ? 

For numeric fields , this does not apply as inverted index / term lookups etc are not relevant.

Reply:
Hi @bharath-techie: Will you please make any necessary changes for technical accuracy? This isn't a topic I'm familiar with so I need some guidance from the SME. Greatly appreciated. Please also see the editor's comment below.